pkg/build/api/types.go

```diff
@@ -86,6 +95,10 @@ type BuildStatus struct {
 	// Cancelled describes if a cancel event was triggered for the build.
 	Cancelled bool
 
+	// Skipped indicates that this build was marked as skipped. The build will be
```

Comment:
skipped is not the same as "not yet ready".  skipped represents a final state.

arguably it *should* be a build state (like completed/failed/etc).  or perhaps we don't need it at all and we can just have the state be "canceled" and the reason would be "skipped due to newer build request"